axelrod/cooperation.py

```diff
@@ -1,7 +1,66 @@
 from . import eigen
+from axelrod import Actions
 
-def cooperation_matrix(results):
+C, D = Actions.C, Actions.D
+
+# As yet unused until RoundRobin returns interactions
+def cooperation_matrix(interactions, nplayers):
```

Comment:
given an `interactions` dict, couldn't the number of players be inferred? Basically solving for `len(dict.keys()) = n*(n-1)/2`